ci/kind/test-clickhouse-migrate.sh

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env bash
```

Comment:
I'm not good at naming. But maybe `test-migrate-clickhouse.sh` will be better considering the consistency?